exchange/websocket/connection.go

```diff
@@ -374,8 +374,8 @@ func (c *connection) Shutdown() error {
 }
 
 // SetURL sets connection URL
-func (c *connection) SetURL(url string) {
-	c.URL = url
+func (c *connection) SetURL(connURL string) {
+	c.URL = connURL
```

Comment:
🐒 We went for just `u` below and I think that's alright
```suggestion
func (c *connection) SetURL(u string) {
	c.URL = u
```